lib/src/Pos/DB/Block.hs

```diff
@@ -496,7 +498,9 @@ getData fp = flip catch handle $ liftIO $
         | otherwise = throwM e
 
 putData ::  (MonadIO m, Bi v) => FilePath -> v -> m ()
-putData fp = liftIO . BS.writeFile fp . serialize'
+putData fp v = liftIO $
+    bracket (openBinaryFile fp WriteMode) hClose $ \h ->
+        BS.hPut h (serialize' v) >> hFlush h
```

Comment:
This whole thing is in `IO`, so you could've used `withBinaryFile` instead (it's in `base`).

Reply:
Not that important, but thanks for the hint.

Reply:
ok, i'll implement it